ios/FluentUI/Vnext/Avatar/AvatarModifiers.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+public extension AvatarView {
+
+    func accessibilityLabel(_ accessibilityLabel: String?) -> AvatarView {
+        state.accessibilityLabel = accessibilityLabel
+        return self
+    }
+
+    func backgroundColor(_ backgroundColor: UIColor?) -> AvatarView {
+        state.backgroundColor = backgroundColor
+        return self
+    }
+
+    func foregroundColor(_ foregroundColor: UIColor?) -> AvatarView {
+        state.foregroundColor = foregroundColor
+        return self
+    }
+
+    func hasPointerInteraction(_ hasPointerInteraction: Bool) -> AvatarView {
+        state.hasPointerInteraction = hasPointerInteraction
+        return self
+    }
+
+    func hasRingInnerGap(_ hasRingInnerGap: Bool) -> AvatarView {
+        state.hasRingInnerGap = hasRingInnerGap
+        return self
+    }
+
+    func image(_ image: UIImage?) -> AvatarView {
```

Comment:
I don't think we should need modifiers for arguments that can be set during `init` (so `style`, `size`, `image`, `primaryText`, and `secondaryText`)

Reply:
Yeah! I agree!